### DISCUSSION TITLE
"Legal info"，Account verification failed

### DISCUSSION THREAD

**Discussion by lixby** [microsoft/MicrosoftEdge-Extensions#97] [Discussions]
1."Legal info"，Account verification failed，The flow is continuously stuck at 'Email verification' and cannot proceed further.
2.We have received the verification email, but after entering the email address, it shows 'There is no orbbec.com in the system, 
please do not enter the incorrect domain', while our account information is indeed using 'develop@orbbec.com'.
![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/17895861/13e0cfb6-64bd-4ffa-84cc-f8d37bb191a4)
![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/17895861/4cc6d39a-cde8-49e6-80d8-7fa6b681c77d)
![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/17895861/845fa815-2059-4220-b6dd-40dd1656dca9)
![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/17895861/c2a0103f-ea0a-4fc5-8838-9e61d22db085)

**Comment by Bhuvana-Priya** [comment]
Hi @lixby,

We are looking into this and will get back to you with further updates.

Thank you
